spec/std/int_spec.cr

```diff
@@ -734,27 +754,32 @@ describe "Int" do
     it { 5_i64.popcount.should eq(2) }
     it { 9223372036854775807_i64.popcount.should eq(63) }
     it { 18446744073709551615_u64.popcount.should eq(64) }
+
+    it { 0_i128.popcount.should eq(0) }
+    # PENDING
+    # it { 170141183460469231731687303715884105727_i128.popcount.should eq(127) }
+    # it { 340282366920938463463374607431768211455_u128.popcount.should eq(128) }
```

Comment:
This doesn't need to be pending. The specs are about `#popcount`, we can just parse the integer at runtime to get the test values. Or just use `Int128::MAX` and `UInt128::MAX`.
```suggestion
    it { Int128::MAX.popcount.should eq(127) }
    it { UInt128::MAX.popcount.should eq(128) }
```